lib/src/main/java/com/orange/ods/compose/component/textfield/OdsOutlinedTextField.kt

```diff
@@ -106,92 +109,85 @@ fun OdsOutlinedTextField(
-        cursorColor = OdsTheme.colors.primary,
-        errorCursorColor = OdsTheme.colors.error,
-        focusedBorderColor = OdsTheme.colors.primary.copy(alpha = ContentAlpha.high),
-        unfocusedBorderColor = OdsTheme.colors.onSurface.copy(alpha = ContentAlpha.disabled),
-        errorBorderColor = OdsTheme.colors.error,
-        leadingIconColor = OdsTheme.colors.onSurface,
-        trailingIconColor = OdsTheme.colors.onSurface,
-        errorTrailingIconColor = OdsTheme.colors.onSurface,
-        focusedLabelColor = OdsTheme.colors.onSurface,
-        unfocusedLabelColor = OdsTheme.colors.onSurface.copy(ContentAlpha.medium),
-        errorLabelColor = OdsTheme.colors.onSurface,
-        placeholderColor = OdsTheme.colors.onSurface.copy(ContentAlpha.medium),
-    )
+                trailingText != null -> {
+                    {
+                        Text(
+                            modifier = Modifier.padding(end = dimensionResource(id = R.dimen.spacing_s)),
+                            text = trailingText,
+                            style = OdsTheme.typography.subtitle1,
+                            color = OdsTextFieldDefaults.trailingTextColor(value.isEmpty(), enabled)
+                        )
+                    }
+                }
+                else -> null
+            },
+            isError = isError,
+            visualTransformation = visualTransformation,
+            keyboardOptions = keyboardOptions,
+            keyboardActions = keyboardActions,
+            singleLine = singleLine,
+            maxLines = maxLines,
+            colors = OdsTextFieldDefaults.outlinedTextFieldColors()
+        )
 
+        OdsTextFieldBottomRow(isError = isError, errorMessage = errorMessage, characterCounter = characterCounter)
+    }
 }
 
 @UiModePreviews.Default
 @Composable
-private fun PreviewOdsOutlinedTextField() = Preview {
+private fun PreviewOdsOutlinedTextField(@PreviewParameter(OdsTextFieldPreviewParameterProvider::class) parameter: OdsTextFieldPreviewParameter) = Preview {
     var value by remember { mutableStateOf("Input text") }
-    OdsOutlinedTextField(
-        value = value,
-        onValueChange = { value = it },
-        placeholder = "Placeholder",
-        leadingIcon = painterResource(id = android.R.drawable.ic_dialog_info),
-        trailingIcon = painterResource(id = android.R.drawable.ic_input_add)
-    )
-}
+    Column {
+        OdsOutlinedTextField(
+            value = value,
+            onValueChange = { value = it },
+            placeholder = "Placeholder",
+            leadingIcon = painterResource(id = android.R.drawable.ic_dialog_info),
+            trailingIcon = painterResource(id = android.R.drawable.ic_input_add),
+            isError = parameter.hasErrorMessage,
+            errorMessage = if (parameter.hasErrorMessage) "Error message" else null
+        )
+
+        if (parameter.hasCounter) {
+            // OdsTextFieldCounter(value.length, 30, Modifier.align(Alignment.End))
```

Comment:
We could display the character counter using the `hasCounter` variable of `OdsTextFieldPreviewParameter` and remove this block of code which is commented anyway.

Same comment for `OdsPasswordOutlinedTextField` and `OdsPasswordTextField`.